e3tools/local-development-portal
# Add preview, download and fix carrousel when opened the second time

## What does this PR do?

Add preview, download and fix carrousel when opened the second time

## How to test?

- [ ] Check that carrousel error is fixed
- Open the modal and close it
- Open the modal by clicking on a different attachment
- Check that the carrousel works as the first you opened

- [ ] Check the preview of PDF
- Open a PDF attachment modal and see a preview

- [ ] Check download of a single PDF
- Open attachment modal
- Check that you can click on the download button and get the file donwloaded 

## Screenshots
![image](https://github.com/e3tools/local-development-portal/assets/16296997/56cd1906-be20-4990-be7e-b52408888424)
![image](https://github.com/e3tools/local-development-portal/assets/16296997/619d1208-b4f4-4fb8-9179-10d1592ab533)
![image](https://github.com/e3tools/local-development-portal/assets/16296997/b3c69da6-9314-4aea-9fc1-67c8d028b062)
